src/facade/reply_builder.cc

```diff
@@ -258,6 +259,7 @@ void MCReplyBuilder::SendLong(long val) {
 
 void MCReplyBuilder::SendError(string_view str, std::string_view type) {
   last_error_ = str;
+  last_error_status_ = OpStatus::CUSTOM_ERROR;
```

Comment:
I don't understand why we need `CUSTOM_ERROR` and why we need the `last_error_status_` in `memcached reply builder` ? From my understanding we are dealing with cluster here so why do we need to change memcached reply builder ? Furthermore, can you plz explain the rationale behind what `CUSTOM_ERROR` means?

Reply:
`CUSTOM_ERROR` is to have status of reported errors that doesn't have status enum code associated with it.